src/components/Navigation/NavLink.component.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { Theme } from '@mui/material';
+import {
+  NavLink as NavLinkBase,
+  NavLinkProps as NavLinkBaseProps,
+} from 'react-router-dom';
+
+type NavLinkProps = NavLinkBaseProps & {
+  theme?: Theme;
+};
+
+export const NavLink = styled(NavLinkBase)(({ theme }: NavLinkProps) => ({
```

Comment:
You could also use the `Link` from mui and pass `NavLinkBase` as a `component` prop. This way it would work as a react router link, and look like the mui one, and you would also have the `sx` prop.

```typescript
import MuiLink from '@mui/material/Link';
import { Link } from 'react-router-dom';

  <MuiLink
    component={Link}
    color="textPrimary"
    to={`${rowData.governorType}/${value}`}
    sx={{ ...
````